src/routes/jira/jira-router.ts

```diff
@@ -31,6 +32,8 @@ JiraRouter.use("/events", JiraEventsRouter);
 
 JiraRouter.get("/", csrfMiddleware, jiraSymmetricJwtMiddleware, jiraAdminPermissionsMiddleware, JiraGet);
 
+JiraRouter.use("/workspaces", JiraWorkspacesRouter);
```

Comment:
Should we do `JiraRouter.use("/workspaces", jiraSymmetricJwtMiddleware, JiraWorkspacesRouter);` here so that all routes inside are covered by `jiraSymmetricJwtMiddleware`?

Reply:
Just following the pattern we're literally using for all other routes we're using here:

```
JiraRouter.use("/connect", JiraConnectRouter);

JiraRouter.post("/sync",
	body("commitsFromDate").optional().isISO8601(),
	returnOnValidationError,
	jiraSymmetricJwtMiddleware,
	JiraSyncPost);

JiraRouter.use("/events", JiraEventsRouter);

...

JiraRouter.use("/workspaces", JiraWorkspacesRouter);

JiraRouter.use("/configuration", JiraConfigurationRouter)
```